doc/classes/AABB.xml

```diff
@@ -18,7 +18,15 @@
 			<argument index="1" name="size" type="Vector3">
 			</argument>
 			<description>
-				Optional constructor, accepts position and size.
+				Constructs an [AABB] from a position and size.
+			</description>
+		</method>
+		<method name="abs">
+			<return type="AABB">
+			</return>
+			<description>
+				Returns an AABB with equivalent position and size, modified so that
+				the most-negative corner is the origin and the size is positive.
```

Comment:
Don't do hard line wraps, one line == one paragraph.
```suggestion
				Returns an AABB with equivalent position and size, modified so that the most-negative corner is the origin and the size is positive.
```